src/main/java/org/opensearch/geospatial/action/upload/geojson/UploadGeoJSONRequest.java

```diff
@@ -14,43 +14,36 @@
 import java.io.IOException;
 import java.util.Objects;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+
 import org.opensearch.action.ActionRequest;
 import org.opensearch.action.ActionRequestValidationException;
 import org.opensearch.common.bytes.BytesReference;
 import org.opensearch.common.io.stream.StreamInput;
 import org.opensearch.common.io.stream.StreamOutput;
 import org.opensearch.rest.RestRequest;
 
+@AllArgsConstructor
+@Getter
 public class UploadGeoJSONRequest extends ActionRequest {
 
+    /**
+     * Request method type. This will be useful to decide whether new index should
+     * be created for upload request or not.
+     */
+    @NonNull
     private final RestRequest.Method method;
+    @NonNull
     private final BytesReference content;
 
-    public UploadGeoJSONRequest(RestRequest.Method method, BytesReference content) {
-        super();
```

Comment:
did we check that super will be called by using the AllArgsConstructor?

Reply:
+1, I think it's not possible for constructor related annotations - https://stackoverflow.com/a/29771875

Reply:
Yeah. it is not possible. However If a constructor does not explicitly invoke a superclass constructor, the Java compiler automatically inserts a call to the no-argument constructor of the superclass.

Reply:
Can you please check delombok source here and confirm this?

Reply:
It works. I double checked with debugger too.